src/fu-tool.c

```diff
@@ -1937,6 +1937,27 @@ fu_util_get_remotes (FuUtilPrivate *priv, gchar **values, GError **error)
 	return TRUE;
 }
 
+static gboolean
+fu_util_get_hsi (FuUtilPrivate *priv, gchar **values, GError **error)
+{
+	g_autoptr(GPtrArray) attrs = NULL;
+	if (!fu_util_start_engine (priv, FU_ENGINE_LOAD_FLAG_NONE, error))
+		return FALSE;
+	attrs = fu_engine_get_host_security_attrs (priv->engine, error);
+	if (attrs == NULL)
+		return FALSE;
+	for (guint i = 0; i < attrs->len; i++) {
+		FwupdHsiAttr *attr = g_ptr_array_index (attrs, i);
+		g_autofree gchar *str = fwupd_hsi_attr_to_string (attr);
+		g_print ("%s\n", str);
+	}
```

Comment:
What do you think about doing tree output here? We're using that everywhere else, so maybe should be considered here too.

Reply:
Right, the `fwupd_hsi_attr_to_string()` was just a hack to show something quickly. If nothing else we need to translate the attribute titles like we do releases and devices already. I guess a tree mode might be prettier. If you want to take this I'll try and implement some more of the tests from the document.

Reply:
This is now pretty.